database/src/main/java/com/firebase/ui/database/FirebaseRecyclerAdapter.java

```diff
@@ -119,12 +120,27 @@ public int getItemCount() {
     }
 
     /**
-     * Method for changing/updating the {@link Query} in existing adapter.
-     *
-     * @param newQuery is a new updated query.
+     * Re-initialize the Adapter with a new set of options. Can be used to change the query
+     * without re-constructing the entire adapter.
      */
-    public void updateQuery(@NonNull Query newQuery) {
-        mSnapshots.updateQuery(newQuery);
+    public void setOptions(@NonNull FirebaseRecyclerOptions<T> options) {
```

Comment:
Can we name it something different? Like 'changeOptions()', 'updateOptions()' or any so it will be easy to recognize this operation as it is setting new `options` in the adapter.